src/core/mavlink_commands.h

```diff
@@ -65,19 +65,25 @@ class MavlinkCommandSender {
         }
     };
 
+#ifdef ARDUPILOT
+#define COMMAND_LONG_PARAM_UNSET 0
+#else
+#define COMMAND_LONG_PARAM_UNSET NAN
+#endif
```

Comment:
I'm not a big fan of this but for now it's ok. I'll revisit it later.

Reply:
> we probably should save the takeoff altitude here
  @julianoes Do you mean to save it in a private variable if it is Ardupilot? Because now it's setting MIS_TAKEOFF_ALT param which is PX4-specific